catalog-rest-service/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -274,57 +274,65 @@
           "description": "Column Name.",
           "type": "string"
         },
-        "valuesCount": {
+        "count": {
           "description": "Total count of the values in this column.",
-          "type": "number"
+          "type": "integer"
         },
         "valuesPercentage": {
           "description": "Percentage of values in this column with respect to rowcount.",
           "type": "number"
         },
         "validCount": {
           "description": "Total count of valid values in this column.",
-          "type": "number"
+          "type": "integer"
         },
         "duplicateCount": {
           "description": "No.of Rows that contain duplicates in a column.",
-          "type": "number"
+          "type": "integer"
         },
         "nullCount": {
           "description": "No.of null values in a column.",
-          "type": "number"
+          "type": "integer"
         },
-        "nullProportion": {
+        "nullRatio": {
           "description": "No.of null value proportion in columns.",
           "type": "number"
         },
-        "missingPercentage": {
+        "missingRatio": {
```

Comment:
If these changes are an issue now, I can revert this and change the naming on the python side. We can tackle it later if needed

Reply:
lets not change it. UI can fix it.